pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -108,8 +108,15 @@ func (q *QueryReconciler) collectInfoFromVdb(ctx context.Context) (ctrl.Result,
 // Temporarily, runListRestorePoints will not call the ListRestorePoints API
```

Comment:
I'm wondering what is `setListRestorePointsQueryConditions` in the above comment